tests/infra_test.go

```diff
@@ -299,14 +299,24 @@ var _ = Describe("[Serial]Infrastructure", func() {
 				// tolerate this taint because it is meant to be used on compute nodes only. If we set this taint
 				// on a master node, we risk in breaking the test cluster.
 				for _, pod := range pods.Items {
-					node := schedulableNodes[pod.Spec.NodeName]
+					node, ok := schedulableNodes[pod.Spec.NodeName]
+					if !ok {
+						// Pod is running on a non-schedulable node?
+						continue
+					}
 					if _, isMaster := node.Labels["node-role.kubernetes.io/master"]; isMaster {
 						continue
 					}
 					selectedNodeName = node.Name
 					break
 				}
 
+				// It is possible to run this test on a cluster that simply does not have worker nodes.
+				// Since KubeVirt can't control that, the only correct action is to halt the test.
+				if selectedNode == nil {
```

Comment:
```
13:40:26: /root/.cache/bazel/_bazel_root/6f347497f91c9a385dcd9294645b76e0/sandbox/processwrapper-sandbox/1517/execroot/__main__/tests/infra_test.go:316:8: undefined: selectedNode
```

The variable is not defined.

Reply:
Ah! that's silly of me. I wrote the patch then realized I needed to sync master and rebase.